go/cmd/vttablet/cli/cli.go

```diff
@@ -116,6 +122,10 @@ func run(cmd *cobra.Command, args []string) error {
 		return err
 	}
 
+	if err := waitForDBAGrants(config, dbaGrantWaitTime); err != nil {
```

Comment:
It will be better to move this into `createTabletServer` versus calling it at the top level. That is the function that eventually calls `NewTabletServer` which depends on dba connections and grants.

Reply:
Done 👍

Reply:
I moved the function into tablet server package too.